unlock-app/src/components/creator/CreatorLocks.js

```diff
@@ -16,7 +16,9 @@ export default class CreatorLocks extends Component {
           <LockMinorHeader>Quantity</LockMinorHeader>
           <LockMinorHeader>Price</LockMinorHeader>
           <LockMinorHeader>Balance / Earnings</LockMinorHeader>
-          <LockHeaderButton>&nbsp;</LockHeaderButton>
+          <LockHeaderButtonContainer>
```

Comment:
But why do you still have `LockHeaderButtonContainer` ?

Reply:
ho I think this is what's causing you issues with `align-self`. Basically this would align the element in the parent... and since you have a element between the 2 that's causing you trouble ;)